src/importexport/musicxml/internal/musicxml/importmxmlpass2.cpp

```diff
@@ -2328,6 +2338,13 @@ void MusicXMLParserPass2::staffDetails(const QString& partId)
         t->setFrets(25);      // sensible default
     }
 
+    QString visible = _e.attributes().value("print-object").toString();
+    if (visible == "no") {
+        _score->staff(staffIdx)->setVisible(false);
+    } else if (visible != "yes") {
+        _logger->logError(QString("print-object should be \"yes\" or \"no\""));
+    }
```

Comment:
What happens if the `print-object` attribute was not specified? Will that also print an error message in the log? I don't think we want it to do that.

Reply:
Yeah, I didn't notice that. I added a check to see if its empty